config/initializers/twofa_encryption_key.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# The 2fa encryption key is used to encrypt users' OTP tokens in the database.
+
+# You can regenerate this key by running `rake generate:twofa_key`
+
+# If you change this key after a user has set up 2fa
+# the users' tokens cannot be recovered
+# and they will not be able to log in again!
+
+Diaspora::Application.config.twofa_encryption_key = '052717f82d2676edef29695037fc21b6eb83661fb40ab5d3fe709cc34095ba08'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.